apps/frontend/src/modules/contributors/slices/actions.ts

```diff
@@ -27,6 +27,16 @@ const loadAll = createAsyncThunk<
 	return await contributorApi.getAll(query);
 });
 
+const loadAllByProjectId = createAsyncThunk<
+	ContributorGetAllResponseDto,
+	{ projectId: string },
```

Comment:
Should `projectId` be a string, not a number?

Reply:
Right. It's better to have this as number. I made a hude refactoring where change the type